dokka-subprojects/plugin-base/src/test/kotlin/expectActuals/ExpectActualsTest.kt

```diff
@@ -9,11 +9,102 @@ import org.jetbrains.dokka.model.withDescendants
 import org.jetbrains.dokka.pages.ClasslikePageNode
 import kotlin.test.Test
 import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
 import kotlin.test.assertTrue
 
 
 class ExpectActualsTest : BaseAbstractTest() {
 
+    private val multiplatformConfiguration = dokkaConfiguration {
+        sourceSets {
+            val commonId = sourceSet {
+                sourceRoots = listOf("src/common/")
+                analysisPlatform = "common"
+                name = "common"
+                displayName = "common"
+            }.value.sourceSetID
+            sourceSet {
+                sourceRoots = listOf("src/jvm/")
+                analysisPlatform = "jvm"
+                name = "jvm"
+                displayName = "jvm"
+                dependentSourceSets = setOf(commonId)
+            }
+            sourceSet {
+                sourceRoots = listOf("src/native/")
+                analysisPlatform = "native"
+                name = "native"
+                displayName = "native"
+                dependentSourceSets = setOf(commonId)
+            }
+        }
+    }
+    private val commonSourceSetId =
+        multiplatformConfiguration.sourceSets.single { it.displayName == "common" }.sourceSetID
+
+    @Test
+    fun `should recognize expect property in class`() = testInline(
```

Comment:
Can you emphasize that a declaration should be without  the `expect` keyword, please?

Reply:
what do you mean? It's not possible to declare `expect val/fun` inside `expect class/interface/etc`